components/FeatureHighlight/src/MDCFeatureHighlightViewController.h

```diff
@@ -122,6 +122,21 @@ typedef void (^MDCFeatureHighlightCompletion)(BOOL accepted);
  */
 @property(nonatomic, strong, nullable) UIColor *bodyColor;
 
+/**
+ Sets the font to be used for the title text.
+
+ Defaults to nil.
+ */
+@property(nonatomic, strong, nullable) UIFont *titleFont;
+
+/**
+ Sets the font to be used for the body text.
+
+ Defaults to nil.
+ */
+@property(nonatomic, strong, nullable) UIFont *bodyFont;
```

Comment:
Pull these new APIs out to a separate PR please so that, if we have to revert this PR, we don't lose all of the functionality.

Reply:
Done. #3274

Reply:
I am waiting for the other PR to get reviewed.

Reply:
Sorry, I think I misread your comment:

> tests are fixed, separated PR has been submitted.